### DISCUSSION TITLE
GitHub pages is not working for me

### DISCUSSION THREAD

**Discussion by Hamza-Noah** [community/community#22026] [New to GitHub]
<p>Hello I really need help I cannot use GitHub pages and I asked here and on stack overflow but I couldn’t get an answer if anyone knows how can I reach the support please help me</p>
<p><a>https://stackoverflow.com/questions/71171324/github-pages-is-not-working-anymore-for-me</a></p>

**Comment by airtower-luna** [comment]
<p>Look at the Actions tab of the repository, you should find logs of Pages trying to set up your site there. If you need more help, please link the repository and describe exactly <em>how</em> it’s “not working”.</p>

**Comment by Hamza-Noah** [comment]
<p>this is the link to the repository <code>https://github.com/Hamza-Noah/New-test</code> my problem is not only on this repository it’s on any repository I create, so if I want to get a GitHub page or live demo I change the branch from none to main and the save button is not clickable</p>

**Comment by Hamza-Noah** [comment]
<p>this is the action page I cannot reach an solution for this problem<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/3/f38cd0fdbb3b4a44f7422f708612a1cbd030b04f.png" data-download-href="/uploads/short-url/yKxIPe4ZN9y2yGI2DUp1upBMLAH.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/82677794/181068324-22316f9b-4786-4d8d-85d8-e9d262645d8c.png" alt="image" data-base62-sha1="yKxIPe4ZN9y2yGI2DUp1upBMLAH" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/3/f38cd0fdbb3b4a44f7422f708612a1cbd030b04f_2_690x388.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/3/f38cd0fdbb3b4a44f7422f708612a1cbd030b04f_2_1035x582.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/f/3/f38cd0fdbb3b4a44f7422f708612a1cbd030b04f_2_1380x776.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/3/f38cd0fdbb3b4a44f7422f708612a1cbd030b04f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1080 129 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
y

**Comment by airtower-luna** [comment]
<p>If enabling Pages in the first place is the problem the Actions won’t help, unfortunately. Usually when people say “don’t work” they mean that something during build or deploy failed, and in that case you’d see the logs there. :wink:</p>

**Comment by Hamza-Noah** [comment]
<p>sorry but I did not get it if you mention any solution for me please explain it to me because this problem is not ending for me</p>

**Comment by airtower-luna** [comment]
<p>I don’t have a solution either, I’m afraid. If the folder selection doesn’t appear after selecting a branch best contact GitHub support directly, so they can check why: <a href="https://support.github.com/request" class="inline-onebox">GitHub Support</a></p>

**Comment by jfullstackdev** [comment]
<p>this was the original post actually,</p>
<aside class="quote quote-modified" data-post="1" data-topic="230097">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/github-pages-is-not-working-for-me-anymore/230097">GitHub pages is not working for me anymore</a> 
  </div>
  <blockquote>
    Hello everyone 
I’ve been used to using GitHub pages to get a live page or live demo for any repository I have created but  it is not working for me anymore on any new repository I go to the GitHub pages section in the repo setting and I change the branch from none to main but  the save button is not clickable on any new repository what should I do to solve this problem 
 <a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/a/5aca5d604a2a61639650a0907627a67d590df0b1.png" data-download-href="/uploads/short-url/cXaxhwXaddGrekAY5mwYieD6HFn.png?dl=1" title="image" rel="noopener nofollow ugc">[image]</a>
  </blockquote>
</aside>

<p>it is really strange, particularly it’s a Pro user type not GitHub Free</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181068329-8704e03a-0983-436c-bc7f-6e6cedbca6fc.png" alt="image" data-base62-sha1="ikpL4fvGq3YVUhQQ4AhYXxJk175" width="273" height="281"></p>

**Comment by Hamza-Noah** [comment]
<p>That’s right I am a pro user. until now I am still facing the problem and I cannot fix it I am trying to contact GitHub support but all I can find is ask on the community here so I came here and saw your comment can you help me reach the support</p>

**Comment by lusrx** [comment]
<aside class="quote no-group" data-username="Hamza-Noah" data-post="9" data-topic="230338">
<div class="title">
<div class="quote-controls"></div>
 Hamza-Noah:</div>
<blockquote>
<p>That’s right I am a pro user.</p>
</blockquote>
</aside>
<p>I can see that you’re a Pro user. Although “Pro” is not one of the price plans.</p>
<p>GitHub has the following <a href="https://github.com/pricing" rel="noopener nofollow ugc">price plans</a>:</p>
<ul>
<li>Free</li>
<li>Team</li>
<li>Enterprise</li>
</ul>
<p>However, according to <a href="https://docs.github.com/en/get-started/learning-about-github/githubs-products#github-pro" rel="noopener nofollow ugc">GitHub Docs</a>, as a Pro user you are entitled to GitHub Support via email.</p>
<p>So I’m not quite sure what all this means, where it leaves you. Do you have access to official support from GitHub or not? I don’t know.</p>
<aside class="quote no-group" data-username="Hamza-Noah" data-post="9" data-topic="230338">
<div class="title">
<div class="quote-controls"></div>
 Hamza-Noah:</div>
<blockquote>
<p>I am trying to contact GitHub support but all I can find is ask on the community here so I came here and saw your comment can you help me reach the support</p>
</blockquote>
</aside>
<p>Have you tried clicking on the “<a href="https://support.github.com/tickets/personal/0" rel="noopener nofollow ugc">My Tickets</a>” link in the upper right corner of the <a href="https://support.github.com/request" rel="noopener nofollow ugc">GitHub Support</a> site and then clicking on the “<a href="https://support.github.com/contact" rel="noopener nofollow ugc">New ticket</a>” button? I only have a Free account and if I click on that button, it only returns me to the landing page of GitHub Support. But since you have a Pro account you might see something else instead of this support maze game. Make sure you’re logged in with the right account in case you have more than one.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/d/7d22d6d1fdced0669d6fab524b9c86dd4e5675ae.png" data-download-href="/uploads/short-url/hR0kenUpUMnlWXZTx0FUAMczKX4.png?dl=1" title="github-support" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/22495331/181068331-2611ae93-461f-4e61-89fe-b2eb80dfacea.png" alt="github-support" data-base62-sha1="hR0kenUpUMnlWXZTx0FUAMczKX4" width="647" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d22d6d1fdced0669d6fab524b9c86dd4e5675ae_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">github-support</span><span class="informations">1003×775 26.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Hamza-Noah** [comment, marked as answer]
<p><a class="mention" href="/jdevstatic">@jdevstatic</a> <a class="mention" href="/airtower-luna">@airtower-luna</a><br>
Thanks for all your help I just want to mention how I solved this problem I was trying everyday searching and asking and trying none of all that worked more than two months and this problem havn’t been solved today I just tried something stupid but it worked for me you I inspectes the <strong>save</strong> button and removed the disabled property I couldnt belive that it worked :joy: :joy: :joy:</p>

**Reply by samiko-io** [reply]
That's an insane solution lol😂 so glad i found you here. Thanks.

**Reply by Hamza-Noah** [reply]
@samiko-io did it work for you

**Reply by wylly3** [reply]
thanks to teammembers for your work!
may i should create a desk with help+savebuttons  XD 
i will check back later . High 5 to ya

**Reply by Hamza-Noah** [reply]
> thanks to teammembers for your work! may i should create a desk with help+savebuttons XD i will check back later . High 5 to ya

I'm waiting

**Comment by Hamza-Noah** [comment]
<p><a class="mention" href="/airtower-luna">@airtower-luna</a> see the prevous comment by me</p>

**Comment by airtower-luna** [comment]
<p>I’m glad you figured it out, but if that wasn’t caused by a browser extension or something I still highly recommend reporting a bug! :sweat_smile:</p>

**Comment by Hamza-Noah** [comment]
<p>I will thanks for your help</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="Hamza-Noah" data-post="11" data-topic="230338">
<div class="title">
<div class="quote-controls"></div>
 Hamza-Noah:</div>
<blockquote>
<p>but it worked for me you I inspectes the <strong>save</strong> button and removed the disabled property I couldnt belive that it worked</p>
</blockquote>
</aside>
<p>what is that exactly if I may ask, because others might have the same struggle,</p>
<p>you mean you were not just clicking the <code>Save</code> button ?</p>

**Comment by Hamza-Noah** [comment]
<p>Hey sorry for answering lately it was only inspecting the save button and removing the disabled attribute from the save button</p>
